klippy/extras/pause_resume.py

```diff
@@ -25,17 +26,25 @@ def get_status(self, eventtime):
         return {
             'is_paused': paused
         }
+    def send_pause_command(self, from_event=False):
+        # This sends the appropriate pause command from an event.
+        if not self.pause_command_sent:
+            if self.v_sd is not None and self.v_sd.is_active():
+                # Printing from virtual sd, run pause command
+                self.sd_paused = True
+                if from_event:
+                    self.v_sd.do_pause_from_event()
+                else:
+                    self.v_sd.cmd_M25({})
```

Comment:
Could this just be `self.v_sd.do_pause()` instead of making the distinction between M25 and do_pause_from_event()?

Reply:
Yep, not a problem.